example/oneLinerCodeExample.html

```diff
@@ -66,13 +66,13 @@
       }).setView([43, -83], 3);
       map.options.minZoom = 3;
 
-      var baselayer1 = L.tileLayer(
-        'https://api.tiles.mapbox.com/v4/mapbox.emerald/{z}/{x}/{y}.png?access_token=pk.eyJ1Ijoic2FnYXJwcmVldDk3IiwiYSI6ImNqaXhlZjgwaDJtb3EzcW1zdDdwMzJkODcifQ.MA2YIv6VpGLLAo-QYUudTA',
-        {
-          attribution:
-                  '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
-        },
-      ).addTo(map);
+      // var baselayer1 = L.tileLayer(
+      //   'https://api.tiles.mapbox.com/v4/mapbox.emerald/{z}/{x}/{y}.png?access_token=pk.eyJ1Ijoic2FnYXJwcmVldDk3IiwiYSI6ImNqaXhlZjgwaDJtb3EzcW1zdDdwMzJkODcifQ.MA2YIv6VpGLLAo-QYUudTA',
+      //   {
+      //     attribution:
+      //             '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
+      //   },
+      // ).addTo(map);
```

Comment:
Let's uncomment this. Thanks

Reply:
I've uncommented these lines on my latest commit. But I have also uncommented lines 83 to 85 so that the layer is added correctly to the map. This makes sure that only the baselayer passed in will be added to the map.